crates/view/src/service.rs

```diff
@@ -1627,4 +1633,11 @@ impl ViewService for ViewServer {
             denom_metadata: metadata.map(Into::into),
         }))
     }
+
+    async fn delegations_by_address_index(
+        &self,
+        _request: tonic::Request<pb::DelegationsByAddressIndexRequest>,
+    ) -> Result<tonic::Response<Self::DelegationsByAddressIndexStream>, tonic::Status> {
+        unimplemented!("delegations_by_address_index")
```

Comment:
Looks like this PR isn't ready yet?

Reply:
Well, the idea is that it could be implemented by other implementations of `ViewService`, such as in the web app. (Note that `fn authorize_and_build` above is also unimplemented.) I had to add this method here or else the Rust compiler would complain that this method was missing.